### PR TITLE
report/log: add lunar start time options :moon:

### PR DESCRIPTION
**Pull request by mguentner**
supplying --luna (e.g. instead of --month), the last full moon
will be taken as the start date of the summary.
The start date of a lunar month is not fixed, some cultures use
the new moon (chinese), some use the full moon (hindu) and some
use the first sight of the crescent (jewish / hebrew).

I implemented the full moon calendar because today is a full moon
and a total lunar eclipse.

My hope is to adjust my work load according the moon cycle.

Peace.

### CODE REVIEW

**Comment by jmaupetit** [comment]
That's a wonderful feature :wink: Can you please fix formatting issues (see travis job output)?

**Comment by mguentner** [comment]
All :green_heart:

File: tests/test_fullmoon.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+from watson.fullmoon import get_last_full_moon
+
+
+def test_with_known_date():
+    aniceday = arrow.Arrow(2018, 7, 27, 10, 51)
+    aniceday_lastMoon = arrow.Arrow(2018, 6, 28, 4, 55)
+
+    aniceday_result = get_last_full_moon(aniceday)
+
+    assert aniceday_result == aniceday_lastMoon
```
Comment (jmaupetit) [inline tests/test_fullmoon.py]: Can you remove the two empty lines above?

File: docs/user-guide/commands.md [hunk 2]
```diff
@@ -6,6 +6,28 @@
...
+    $ watson add --from "2018-03-20 12:00:00" --to "2018-03-20 13:00:00" \
+     programming +addfeature
+
+### Options
+
+Flag | Help
+-----|-----
+`-f, --from DATE` | Date and time of start of tracked activity  [required]
+`-t, --to DATE` | Date and time of end of tracked activity  [required]
+`--help` | Show this message and exit.
```
Comment (jmaupetit) [inline docs/user-guide/commands.md]: Looks like this should not be part of this PR, but was missing...

**Comment by mguentner** [comment]
@jmaupetit Done. I will squash a last time before you merge.

**Comment by mguentner** [comment]
@jmaupetit @k4nar squashed and still green
